templates/config_parameter_table.html

```diff
@@ -0,0 +1,179 @@
+<!-- Parameter/arguments table -->
+<div class="eagleTableDisplay" id="parameterTable" tabindex="-1" role="dialog" aria-labelledby="parameterTableTitle">
```

Comment:
Some of these ids (for example: "parameterTableTitle") are re-used in both parameter_table.html and the new config_parameter_table.html.

It is probably best to make the ids unique, even though in practice only one of the elements should be used at any one time.